etc/dbus-serialbattery/battery.py

```diff
@@ -269,7 +270,10 @@ def manage_charge_voltage_linear(self) -> None:
             if self.max_voltage_start_time is None:
                 # start timer, if max voltage is reached and cells are balanced
                 if (
-                    self.max_battery_voltage - utils.VOLTAGE_DROP <= voltageSum
+                    self.max_battery_voltage
+                    - utils.VOLTAGE_DROP
+                    - measurementToleranceVariation
```

Comment:
I don't think that this should be added here, else the problem is only moved to a lower value.

I my opinion the correct should be e.g. start at 55.2, but reset only, if going below 55.1785 (55.2 - `measurementToleranceVariation`).

Reply:
i agree. the first one ist not necessary. you are right.